docs/modules/ROOT/pages/release-notes/release-notes-28-1.adoc

```diff
@@ -0,0 +1,27 @@
+= Payara Enterprise 5.28.1 Release Notes
+
+== Supported APIs and Applications
+
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 3.3
+
+== Notes
+
+=== Incompatibility with the latest JVM / JDK versions and Monitoring
+IMPORTANT: Payara Platform products do run correctly on JDK 8u292, 11.0.11 or later. Due to removal of SSL/TLS 1.0 and 1.1 protocols, monitoring data will not be available because the JMX service fails to start, and will not produce monitoring data. Other issues due to the missing protocols may also be present.
```

Comment:
```suggestion
IMPORTANT: Due to removal of SSL/TLS 1.0 and 1.1 protocols in the latest OpenJDK versions, monitoring data will not be available on JDK 8u292 and 11.0.11 or later because the JMX service fails to start and will not produce monitoring data. Other issues due to the missing protocols may also be present.
```